_tags/iteration.md

```diff
@@ -377,3 +424,40 @@ Defines a range of numbers to loop through. The range can be defined by both lit
 </table>
 {% endraw %}
 ```
+
+## tablerowloop (object)
+
+Information about a parent [`tablerow` loop](#tablerow).
+
+```json
+{
+  "col": 1,
+  "col0": 0,
+  "col_first": true,
+  "col_last": false,
+  "first": true,
+  "index": 1,
+  "index0": 0,
+  "last": false,
+  "length": 5,
+  "rindex": 5,
+  "rindex0": 4,
+  "row": 1
+}
+```
+
+### tablerowloop (properties)
+
+| Property | Description | Returns |
+| `col` | The 1-based index of the current column. | `number` |
+| `col0` | The 0-based index of the current column. | `number` |
+| `col_first` | Returns `true` if the current column is the first in the row. Returns `false` if not. | `boolean` |
```

Comment:
Boolean fields can also use the "`Whether`..." start to the sentences here -- totally up to you!